qiskit_ibm/accountprovider.py

```diff
@@ -220,6 +231,14 @@ def _discover_remote_backends(self, timeout: Optional[float] = None) -> Dict[str
 
         return ret
 
+    def get_backend(self, name: str = None, **kwargs: Dict[str, Any]) -> Union[IBMQBackend, Any]:
+        try:
+            return self.__backends[name]
```

Comment:
So if the backend has already been initialized, this just assumes a `name` is specified and ignore `kwargs`?

Reply:
Updated! The logic should now be corrected. It will request already-loaded backends using name + kwargs, then query from the API if none exist. I added a note that this query can only be done via `name` (not other props). In this case, not providing `name` will cause all backends to be loaded.